ava-labs/core-mobile
# CP-8647 Display X-Chain recent activity

## Description

**Ticket: [CP-8647]** 

Implements displaying X-Chain recent activity for currently active account.
### ! NOTE !
X and P Chain transactions doesn't really show how much amount got or left account, it shows the amount moved with transaction (sum of AVAX in input UTXOs). This means we have incorrect display by showing positive and negative amounts as well as icons for transactions.
That should be fixed either in this or future PRs.

## Screenshots/Videos

![Simulator Screenshot - iPhone 15 - 2024-05-21 at 17 24 10](https://github.com/ava-labs/avalanche-wallet-apps/assets/6573904/bca1d3e3-0a29-4313-b045-8a2137857766)

## Testing
- Switch to X chain network
- Open X chain card on portfolio screen
- switch to Activity tab
= observe list of recent activity

## Checklist

Please check all that apply (if applicable)


[CP-8647]: https://ava-labs.atlassian.net/browse/CP-8647?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
